dnf/base.py

```diff
@@ -466,6 +466,7 @@ def close(self):
         self._closed = True
         self._finalize_base()
         self.reset(sack=True, repos=True, goal=True)
+        self._plugins = None
```

Comment:
The way the `close()`, `reset()` and `_finalize_base()` methods are called is a bit peculiar, but it would appear that setting `self._plugins` to `None` should be put into the `_finalize_base()` method, which does the dirty work, `close()` seems to have been meant just to manage the "closed" state and call the other two methods.

Reply:
I have moved it there.